src/shared/components/MarkDownEditor/MarkDownEditor.utils.ts

```diff
@@ -1,38 +1,27 @@
-import MdEditor from 'md-editor-rt';
+import { config } from 'md-editor-rt';
+
+import emoji from 'markdown-it-emoji';
+import sub from 'markdown-it-sub';
+import sup from 'markdown-it-sup';
+import deflist from 'markdown-it-deflist';
+import abbr from 'markdown-it-abbr';
+import footnote from 'markdown-it-footnote';
+import insert from 'markdown-it-ins';
+import mark from 'markdown-it-mark';
+import taskLists from 'markdown-it-task-lists';
+import container from 'markdown-it-container';
+import katex from 'markdown-it-katex-external';
+import miip from 'markdown-it-images-preview';
+import html5Embed from 'markdown-it-html5-embed';
+import markdownItImSize from 'markdown-it-imsize';
 
 import i18n from 'i18n';
-import { VALID_VIDEO_FILE_TYPES, VALID_AUDIO_FILE_TYPES } from 'shared/consts';
 
-import {
-  ALIGN_RULE,
-  LANGUAGE_BY_DEFAULT,
-  VIDEO_LINK_REGEX,
-  VideoSourcePlayerLinks,
-  VideoSources,
-  YOUTUBE_VIDEO_ID_REGEX,
-} from './MarkDownEditor.const';
+import { LANGUAGE_BY_DEFAULT } from './MarkDownEditor.const';
 
 const { t } = i18n;
 
-export const getValidMediaFormatsRegex = (validFileTypesArray: string[]) => {
```

Comment:
Does .vebm, .mp4, youtube, vimeo video have correct display in the right column after changes?